src/test/java/uk/co/ramp/OutbreakTest.java

```diff
@@ -23,50 +18,40 @@
 import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.test.util.ReflectionTestUtils;
-import uk.co.ramp.event.*;
-import uk.co.ramp.event.types.*;
+import uk.co.ramp.event.CompletionEventListGroup;
+import uk.co.ramp.event.types.ContactEvent;
+import uk.co.ramp.event.types.ImmutableContactEvent;
 import uk.co.ramp.io.InitialCaseReader;
 import uk.co.ramp.io.types.CmptRecord;
 import uk.co.ramp.io.types.DiseaseProperties;
 import uk.co.ramp.io.types.StandardProperties;
 import uk.co.ramp.people.Case;
 import uk.co.ramp.people.Human;
-import uk.co.ramp.policy.alert.TracingPolicyContext;
-import uk.co.ramp.policy.isolation.IsolationPolicyContext;
+import uk.co.ramp.statistics.StatisticsWriter;
 
 @SuppressWarnings("unchecked")
 @DirtiesContext
 @RunWith(SpringRunner.class)
-@Import({
-  TestConfig.class,
-  TestUtils.class,
-  AppConfig.class,
-  IsolationPolicyContext.class,
-  TracingPolicyContext.class
-})
+@Import({TestConfig.class, AppConfig.class})
 public class OutbreakTest {
 
   private final Random random = TestUtils.getRandom();
 
   @Rule public LogSpy logSpy = new LogSpy();
 
+  @Autowired Population population;
   @Autowired private StandardProperties standardProperties;
-
   @Autowired private InitialCaseReader initialCaseReader;
-
   @Autowired private Outbreak outbreak;
-
   @Autowired private CompletionEventListGroup eventListGroup;
 
-  @Autowired Population population;
-
   private DiseaseProperties diseaseProperties;
 
-  public OutbreakTest() throws FileNotFoundException {}
-
   @Before
   public void setUp() throws FileNotFoundException {
     this.diseaseProperties = TestUtils.diseaseProperties();
+    StatisticsWriter statisticsWriter = mock(StatisticsWriter.class);
+    ReflectionTestUtils.setField(outbreak, "statisticsWriter", statisticsWriter);
```

Comment:
It is not necessary to mock the StatisticsWriter here. In my view this test is more of an integration/high-level style test. It uses autowired dependencies, so ends up testing the general functionality of many classes. I would suggest removing these two lines.